### DISCUSSION TITLE
Will Clipping masks ever get added?

### DISCUSSION THREAD

**Discussion by MudkipWorld** [Orama-Interactive/Pixelorama#1000] [Q&A]
Hi, I was digging through the code and found this 
![image](https://github.com/Orama-Interactive/Pixelorama/assets/94318023/a0ceed94-9a92-4633-bca1-384af3fe04a7)
Was this a leftover from a planned feature related to clipping masks (Abandoned) or will it get added in a future release?
This would be super helpful and makes things.

**Comment by Variable-ind** [comment]
> Hi, I was digging through the code and found this ![image](https://private-user-images.githubusercontent.com/94318023/317935625-a0ceed94-9a92-4633-bca1-384af3fe04a7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE2ODg5NTIsIm5iZiI6MTcxMTY4ODY1MiwicGF0aCI6Ii85NDMxODAyMy8zMTc5MzU2MjUtYTBjZWVkOTQtOWE5Mi00NjMzLWJjYTEtMzg0YWYzZmUwNGE3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI5VDA1MDQxMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQwZGQ4NmQzMGE0MjI4MmEyZDYzMTBmY2NhMGU3YzQ5NGE2YjkzYjI2NWJiOTBiZmYwNGJhNmI4NTE2MDlhNzEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.oswMpEB0u5D0w4pYd510keVh5ZEIxsOlXz5rWtQ2mC0) Was this a leftover from a planned feature related to clipping masks (Abandoned) or will it get added in a future release? This would be super helpful and makes things.

Hy, this feature is recently added to pixelorama 1.0
https://github.com/Orama-Interactive/Pixelorama/commit/c1b78e4c01b46edb0e7d58a06995079e0e0db05d
